content/blog/2021/announcing-keynote-speaker-sebastian.md

```diff
@@ -0,0 +1,28 @@
+Title:Announcing our second Keynote Speaker for PyCon India 2021 - Sebastián Ramírez
+Date: 2021-06-02 17:00
+Category: 2021
+Slug: keynote-announcement-sebastian-ramirez
+Summary: We are happy to announce **Sebastián Ramírez** as a Keynote Speaker for PyCon India 2021!
+
+![Sebastián Ramírez](https://in.pycon.org/2021/assets/images/s2.png)
```

Comment:
image link incorrect. replace `s2.png` with `sebastian.png`

Reply:
@vipulgupta2048 ^

Reply:
Yeah, I was flying blind on that one and couldn't find the PyCon Website PR when I wrote this. Good catch. Made the change.